pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -83,6 +86,66 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+type TabPanelProps = {
+  children: any
+  value: any
+  index: any
+}
+
+function TabPanel(props: TabPanelProps) {
+  return (
+      <Typography
+          component="div"
+          role="tabpanel"
+          hidden={props.value !== props.index}
+          id={`scrollable-auto-tabpanel-${props.index}`}
+          aria-labelledby={`scrollable-auto-tab-${props.index}`}
+      >
+        <Box p={3}>{props.children}</Box>
+      </Typography>
+  );
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired
+};
+
+
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+export const ApplicationFormTabs: React.FC<ApplicationFormProps> = (props) => {
+  const [value, setValue] = useState(0);
```

Comment:
nits, looks like this `value` is used to determine if this tab is selected or not. Not a convention, but in that case, instead of using a generic name like `value`, `selectedTabIndex` is more readable.

Reply:
Like it 👍